data/notices.json

```diff
@@ -727,6 +727,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(cli): may fail on credentials coming from credential provider plugins",
+      "issueNumber": 32653,
+      "overview": "Credential Providers returning empty credentials, or credentials with null expiration fields, would cause the CLI to throw an exception from CDK versions 2.172.0 until 2.173.1.",
+      "components": [
+        {
+          "name": "cli",
+          "version": ">=2.172.0 <= 2.173.1"
```

Comment:
Are you sure `2.172.0` introduced this? the issue doesn't mention which PR caused the issue - can you verify and it to the issue?

Reply:
Ok. Please add this to the issue itself 👍

Reply:
WIll do!

Reply:
donezo